pkg/webhooks/resource/handlers.go

```diff
@@ -181,6 +181,11 @@ func (h *handlers) Mutate(logger logr.Logger, request *admissionv1.AdmissionRequ
 		return admissionutils.ResponseFailure(err.Error())
 	}
 	newRequest := patchRequest(mutatePatches, request, logger)
+	policyContext, err = h.pcBuilder.Build(newRequest, mutatePolicies...)
```

Comment:
@shahpratikr - please add a comment to state why we need to rebuild the policy context here e.g.

```go
// rebuild context to process images updated via mutate policies 
```

Reply:
thanks for review. added the comment.